rollup.config.js

```diff
@@ -29,13 +28,12 @@ module.exports = [
 			resolve(),
 			commonjs(),
 			babel(),
-			stylesheet({
```

Comment:
Maybe we should add something to the migration guide about the stylesheet and `disableCSSInjection` flag no longer being available?